src/rotten-tomatoes/index.js

```diff
@@ -1,3 +1,80 @@
-function rottenTomatoes(grid) {}
+function rottenTomatoes(grid) {
+    let gridCopy = grid
+
+  // An array that holds added values to the four adjecent nodes
+  // top, bottom, left and right respectively.
+    const dimensions = [
+      [-1, 0],
+      [1, 0],
+      [0, -1],
+      [0, 1]
+    ];
```

Comment:
Why is this defaulted to these values. I don't understand. What is `-1`, `0`, `1`?

Reply:
I used it to check the four adjacent directions. It's the just the way I leant to solve my own grid problems. 😁